specs/pvs/corla.pvs

```diff
@@ -1133,6 +1216,13 @@ BEGIN
   random_list_of_ballots:
```

Comment:
line 1113/1196 The system we are delivering will use only ballot_permutation, as far as I know. So we should not include the other here. Why confuse the reader?